packaging/kickstart/README.md

```diff
@@ -0,0 +1,172 @@
+  command in kickstart.
+* `MICROSHIFT_VER` references the MicroShift version to install using the `4.x`
+  format. Note that the latest available `.y` version will be installed.
+
+Example commands setting the variables.
+
+```bash
+export RHSM_ORG="$(cat ~/.rhsm-activation-org)"
+export RHSM_KEY="$(cat ~/.rhsm-activation-key)"
+export MICROSHIFT_VER=4.17
+```
+
+Run the following command to create the `kickstart.ks` file to be used during
+the virtual machine installation.
+
+```bash
+envsubst < \
+    /usr/share/microshift/kickstart/kickstart-rpm.ks.template > \
+    "${HOME}/kickstart.ks"
+```
+
+### Image Mode
+
+The following variables need to be added for creating an Image Mode kickstart file.
+
+* `BOOTC_IMAGE_URL` contains a reference to the image to be installed using the
+  [ostreecontainer](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer) kickstart command.
+* `AUTH_CONFIG` contents are copied to `/etc/ostree/auth.json` to authenticate
+  access to the `BOOTC_IMAGE_URL` image. If no registry authentication is required,
+  skip this setting.
+* `REGISTRY_CONFIG` contents are copied to `/etc/containers/registries.conf.d/999-microshift-registry.conf`
+  to configure access to the registry containing the `BOOTC_IMAGE_URL` image.
+  If no registry configuration is required, skip this setting.
+
+Example commands setting the variables.
+
+```bash
+export BOOTC_IMAGE_URL=quay.io/myorg/mypath/microshift-image:tag
+export AUTH_CONFIG="$(cat ~/.quay-auth.json)"
+export REGISTRY_CONFIG="$(cat ~/.quay-config.conf)"
+```
+
+Run the following command to create the `kickstart.ks` file to be used during
+the virtual machine installation.
+
+```bash
+envsubst < \
+    /usr/share/microshift/kickstart/kickstart-bootc.ks.template > \
+    "${HOME}/kickstart.ks"
+```
+
+### OSTree
+
+The following variables need to be added for creating an OSTree kickstart file.
+
+* `OSTREE_SERVER_URL` contains an OSTree server URL passed to the
+  [ostreesetup](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreesetup) kickstart command.
+* `OSTREE_COMMIT_REF` contains an OSTree commit reference to be installed from
+  the server.
+* `AUTH_CONFIG` contents are copied to `/etc/ostree/auth.json` at the pre-install
+  stage to authenticate access to the `OSTREE_SERVER_URL` server. If no server
+  authentication is required, skip this setting.
+
+Example commands setting the variables.
+
+```bash
+export OSTREE_SERVER_URL="<http://my_ostree_server_url>"
+export OSTREE_COMMIT_REF="myostree_commit_reference"
+export AUTH_CONFIG="$(cat ~/.ostree-auth.json)"
+```
+
+Run the following command to create the `kickstart.ks` file to be used during
+the virtual machine installation.
+
+```bash
+envsubst < \
+    /usr/share/microshift/kickstart/kickstart-ostree.ks.template > \
+    "${HOME}/kickstart.ks"
+```
+
+## Create Virtual Machine
+
+Download a RHEL boot ISO image from https://developers.redhat.com/products/rhel/download.
```

Comment:
Would https://access.redhat.com/downloads/content/rhel be more generic link? Is developers portal accessible with customer subscription?

Reply:
Uh, not 100%, worst case customer with sub gets an addition developer sub when using the developer. link.
How about: we use the developer... link for the upstream/community doc, and the access.... link for the enterprise doc? Because that would match quite nice the intention of the two different sites.

Reply:
Yeap, that's a good point. The current doc is more upstream-oriented.